src/attached_probe.cpp

```diff
@@ -147,6 +148,22 @@ int AttachedProbe::detach_iter(void)
   return 0;
 }
 
+void AttachedProbe::attach_raw_tracepoint(void)
+{
+  if (progfd_ < 0)
+    // Errors for raw_tracepoint are handled in load_prog, ignore here
+    return;
+  tracing_fd_ = bpf_raw_tracepoint_open(probe_.attach_point.c_str(), progfd_);
+  if (tracing_fd_ < 0)
+    throw std::runtime_error("Error attaching probe: " + probe_.name);
```

Comment:
We could detect tracepoint existence here:
```suggestion
  if (tracing_fd_ < 0)
  {
    if (tracing_fd_ == -ENOENT)
      throw std::runtime_error("Probe does not exist: " + probe_.name);
    else
      throw std::runtime_error("Error attaching probe: " + probe_.name);
  }
```